t/cli/plugin/test_file_logger_std_output.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
```

Comment:
Is this test run in the CI?
https://github.com/apache/apisix/blob/d019da018ca8c06bf80e363a0512a30808da1b8e/ci/linux_apisix_current_luarocks_runner.sh#L73